lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -2017,7 +1888,77 @@ def each_host_range_chunk(host_ranges, &block)
             end
           end
 
+          #######
           private
+          #######
+
+          def add_data_service(*args)
+            protocol = "http"
+            port = 80
```

Comment:
While we are making changes here it might be nice to update this to use the same default port 8080 as `msfdb` (now `msfdb_ws`).

Reply:
word